tools/deviceGen/output/features_intersect.h

```diff
@@ -0,0 +1,78 @@
+   // VK 1.0 Core
+    res.robustBufferAccess &= _rhs.robustBufferAccess;
+    res.geometryShader &= _rhs.geometryShader;
+    res.tessellationShader &= _rhs.tessellationShader;
+    res.depthBounds &= _rhs.depthBounds;
+    res.wideLines &= _rhs.wideLines;
```

Comment:
I'd appreciate a newline between different groups (and when there is no group, treat like a group with just one item)